x-pack/plugin/transform/qa/multi-node-tests/build.gradle

```diff
@@ -13,17 +13,17 @@ File keystoreDir = new File(project.buildDir, 'keystore')
 File nodeKey = file("$keystoreDir/testnode.pem")
 File nodeCert = file("$keystoreDir/testnode.crt")
 // Add key and certs to test classpath: it expects it there
-task copyKeyCerts(type: Copy) {
+tasks.register("copyKeyCerts", Copy) {
   from(project(':x-pack:plugin:core').file('src/test/resources/org/elasticsearch/xpack/security/transport/ssl/certs/simple/')) {
     include 'testnode.crt', 'testnode.pem'
   }
   into keystoreDir
 }
 // Add keys and cets to test classpath: it expects it there
 sourceSets.test.resources.srcDir(keystoreDir)
-processTestResources.dependsOn(copyKeyCerts)
+processTestResources.dependsOn("copyKeyCerts")
```

Comment:
Should we defer the creation of `processTestResources` here?

Reply:
done